src/applications/disability-benefits/all-claims/tests/schema/transformedData.js

```diff
@@ -84,13 +84,6 @@ export const transformedMaximalData = {
         to: '2020-01-01',
       },
     },
-    additionalDocuments: [
```

Comment:
Removed all of the document properties after flattening them onto `attachments` pretty sure we don't need them to hit the API?